openfisca_canada_mvohwr/variables/standard_hours/standard_hours_weekly.py

```diff
@@ -98,11 +99,20 @@ def formula(persons, period, parameters):
         return not_(persons("work_category__is_highway_operator_only",period)) &\
             (persons("work_category__is_city_operator_only",period))
 
-class standard_hours__is_hmvo_weekly_mvo_hours_of_work(Variable):
+class standard_hours__is_hmvo_weekly_only(Variable):
+    value_type = bool
+    entity = Person
+    label = u"Placeholder"
+    definition_period = DAY
+    reference = u"TODO"
+    def formula(persons, period, parameters):
+        return persons("work_category__is_highway_operator_only",period) 
+
+class standard_hours__is_hmvo_daily_only(Variable):
     value_type = bool
     entity = Person
     label = u"Is HMVO only with standard hours of work per week"
     definition_period = DAY
     reference = u"https://laws-lois.justice.gc.ca/eng/regulations/C.R.C.,_c._990/page-1.html#h-604501"
     def formula(persons, period, parameters):
-        return persons("work_category__is_highway_operator_only",period) 
\ No newline at end of file
+        return persons("work_category__daily_is_highway_operator_only",period) 
```

Comment:
This breaks the weekly calc since we care more about which category had the most hours and not if they only have hours